src/aux-server/aux-web/shared/scene/SceneUtils.ts

```diff
@@ -501,40 +501,30 @@ export function disposeMesh(
 export function disposeObject3D(
     object3d: Object3D,
     disposeGeometry: boolean = true,
-    disposeMaterial: boolean = true
+    disposeMat: boolean = true,
+    disposeTextures: boolean = false
 ) {
-    if (!object3d) return;
-
-    if (disposeGeometry) {
-        let geometry = (<any>object3d).geometry;
-        if (geometry) {
-            geometry.dispose();
-        }
-    }
-
-    if (disposeMaterial) {
-        let material = (<any>object3d).material;
-        if (material) {
-            if (Array.isArray(material)) {
-                for (let i = 0; i < material.length; i++) {
-                    material[i].dispose();
-                }
-            } else {
-                material.dispose();
-            }
-        }
+    if (object3d instanceof Mesh) {
```

Comment:
@Blitzy This doesn't look like it would support other types of objects that have geometry & materials, like `LineSegments`, `Box3Helper`, or `LineHelper`.